package.json

```diff
@@ -14,9 +14,9 @@
   },
   "dependencies": {
```

Comment:
Check if "vue-debounce-decorator" (line 72) is still used by other components in this repo. If not then please uninstall it.

Reply:
- There is one more that uses this. 
https://github.com/bcgov/namerequest/blob/ef3d33ccac9844200360816ad31c4e592aa4b1b1/src/mixins/action-mixin.ts#L130C1-L133

If we need to remove this too, I will do

Reply:
No, let's call this out of scope.

Resolving.